app/code/Edu/Ultimate/Setup/UpgradeData.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Edu\Ultimate\Setup;
```

Comment:
I get that the namespace is Edu, but what is "Ultimate" name of the module? Name should be representative of what module does

Reply:
https://github.com/Maxdzy/eduMagenta2/tree/EDU270-2/app/code/Edu/CmsCreatePage